lib/chef_actions.go

```diff
@@ -186,9 +186,14 @@ func (ar *actionRequest) randomize() {
 	case environmentAction:
 	case roleAction:
 	case policyfileAction:
+	case groupAction:
+	case organizationAction:
+	case permissionAction:
+	case userAction:
+	case itemAction:
```

Comment:
For the itemAction we will want to set the `parent_type` to be bag like the below example message. 
https://github.com/chef/a2/blob/master/components/ingest-service/examples/actions/item_bag_create.json#L8

This is the same for `permissionAction` - https://github.com/chef/a2/blob/master/components/ingest-service/examples/actions/permission_update_container.json